_includes/head.html

```diff
@@ -65,4 +65,5 @@
 
 <!-- Scripts
 ================================================== -->
-  <script src="{{ site.baseurl }}/assets/js/vendor/jquery-1.11.3.min.js"></script>
\ No newline at end of file
+  <script src="{{ site.baseurl }}/assets/js/vendor/jquery-1.11.3.min.js"></script>
```

Comment:
Not sure, I didn't do it. Happy to move it though. Also I'm updating a few script things in a different PR so I can include the move there too. Qq: would this be the first in our scripts?